infra/buckets/s3.tf

```diff
@@ -173,3 +173,17 @@ resource aws_s3_bucket dss_s3_events_bucket {
   }
   tags = merge(local.common_tags, local.aws_tags)
 }
+
+resource aws_s3_bucket dss_cli_bucket_test {
+  count = var.DSS_DEPLOYMENT_STAGE == "dev" ? 1 : 0
+  bucket = var.DSS_CLI_BUCKET_TEST
+  lifecycle_rule {
```

Comment:
I copied this `lifecycle_rule` from the DSS test buckets, but could use a second opinion on whether this is the right lifecycle rule to apply...

Reply:
i think this should be ok, unless there are fixtures that are going to be placed inside the cli-bucket-test, but i think all the fixtures for that repo are within the tests dir.